rootfs/etc/profile.d/_20-mounts.sh

```
@@ -0,0 +1,206 @@
+		local src dest
+		IFS='>' read -r dest src <<<"$l"
+		if [[ -z $src ]] || [[ -z $dest ]]; then
+			red "# ERROR: Invalid symlink definition: $l"
+			continue
+		fi
+		mkdir -p "$(dirname "$dest")"
+		if [[ -e $dest ]]; then
+			red "# ERROR: Symlink destination already exists: '$dest'"
+			continue
+		fi
+		ln -sT "$src" "$dest" && yellow symlinking "'$src' -> '$dest'" || red "# ERROR: Failed to create symlink: '$src' -> '$dest'"
```

Shellcheck (0.10.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)